modules/migration-mtc-release-notes-1-8.adoc

```diff
@@ -0,0 +1,73 @@
+
+// Module included in the following assemblies:
+//
+// * migration_toolkit_for_containers/mtc-release-notes.adoc
+:_content-type: REFERENCE
+[id="migration-mtc-release-notes-1-8_{context}"]
+= {mtc-full} 1.8 release notes
+
+[id="new-features-and-enhancements-1-8{context}"]
+== New features and enhancements
+
+This release has the following new features and enhancements:
+
+This release has no new features and enhancements.
+
+[id="deprecated-features-1-8_{context}"]
+== Deprecated features
+
+The following features are deprecated:
+
```

Comment:
Same with line 19. 

@anarnold97 -- Do we need "new features" and "deprecated features" sections if there are none or do users expect to see these sections? Known and resolved issues are expected, so I can see including those sections even if there are no such issues.  Do you have a boilerplate paragraph for container upgrade releases, such as "This release is a container upgrade release. As such it has neither new features or deprecated features." ?

Reply:
I would remove Line 16 to 19. However, this is not a CGO

Reply:
@anarnold97 Ah, not sure why i thought it was a CGO...my mistake.

Reply:
Welcome to my world! LOL

Reply:
I removed the Deprecated features section.